source/develop/release-management/features/virt/KvmToOvirt.html.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Importing Libvirt KVM VMs to oVirt
+category: feature
+authors: Shahar Havivi
+wiki_category: Feature|v2v
+wiki_title: Features/KvmToOvirt
+wiki_revision_count: 1
+wiki_last_updated: 2017-03-20
+feature_name: 'v2v: Importing Libvirt Kvm Vms to oVirt'
+feature_modules: all
+feature_status: Released in oVirt 4.1
+---
+
+# Importing Libvirt KVM VMs to oVirt
+
+## Summary
+Import KVM based VMs from Libvirt management via the oVirt admin portal.
+There is no need for disk conversion since both oVirt and Libvirt support KVM based VMs.
+Importing VMs with file and block device disks is supported, snapshots are currently not supported. Please note that if you want to import VM with snapshots the current snapshot is the one that will be imported.
```

Comment:
Suggest:
"Disk conversion is not required since both oVirt and Libvirt support KVM-based VMs. 
You can import VMs with file and and block device disks. Snapshots are currently not supported. Note that if you want to import VMs with snapshots, it is the current snapshot that will be imported."

Comment: If snapshots are not supported, how can they be imported?

Reply:
Ad comment: It does not mean you cannot import VM that has snapshots. You can. It's just that resulting VM will not have all the states of the original VM. It will have only single state that corresponds to the current state of the original VM.

Does this make more sense to you?